app/main.py

```diff
@@ -1,8 +1,52 @@
 class Car:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            comfort_class: int,
+            clean_mark: int,
+            brand : str
+    ) -> None:
+        self.comfort_class = comfort_class
+        self.clean_mark = clean_mark
+        self.brand = brand
 
 
 class CarWashStation:
-    # write your code here
-    pass
+    def __init__(
+            self,
+            distance_from_city_center: float,
+            clean_power: int,
+            average_rating: float,
+            count_of_ratings: int
+    ) -> None:
+        self.distance_from_city_center = distance_from_city_center
+        self.clean_power = clean_power
+        self.average_rating = average_rating
+        self.count_of_ratings = count_of_ratings
+
+    def calculate_washing_price(self, car: Car) -> float:
+        return round(
+            car.comfort_class
+            * (self.clean_power - car.clean_mark)
+            * self.average_rating
+            / self.distance_from_city_center,
+            1
+        )
+
+    def serve_cars(self, cars: list[Car]) -> float:
+        total_price = 0.0
+        for car in cars:
+            if car.clean_mark < self.clean_power:
```

Comment:
The condition `if car.clean_mark < self.clean_power:` does not account for the case where `car.clean_mark` is equal to `self.clean_power`. If the intention is to only wash cars that are less clean than the station's clean power, this is correct. Otherwise, consider adjusting the condition.